Source/DiabloUI/selhero.cpp

```diff
@@ -299,18 +299,19 @@ void SelheroClassSelectorEsc()
 
 void SelheroNameSelect(int /*value*/)
 {
-	if (!UiValidPlayerName(selhero_heroInfo.name)) {
+	// only check names in multiplayer, we don't care about them in single
+	if (selhero_isMultiPlayer && !UiValidPlayerName(selhero_heroInfo.name)) { 
 		ArtBackground.Unload();
 		UiSelOkDialog(title, _("Invalid name. A name cannot contain spaces, reserved characters, or reserved words.\n"), false);
 		LoadBackgroundArt("ui_art\\selhero.pcx");
+	} else{

```

Comment:
`} else {`

Reply:
my formatting in visual studio died for some reason :(